app/Console/Commands/LdapSync.php

```diff
@@ -216,6 +216,14 @@ public function handle()
                 $user->country = $item['country'];
                 $user->department_id = $department->id;
 
+                if($item->ldap_default_group != null){
+
+                    $permissions_array = $item['ldap_default_group'];
+                    $user->permissions= json_encode($permissions_array);
```

Comment:
I don't think we need this here? We don't need to decode permissions, just assign them the group that was picked.

Reply:
fixed it.